templates/issue.html

```diff
@@ -283,6 +283,14 @@
 			</div>
 		</div>
 		<div>
+			<div class="participants"> 
+				<div class="number-participants">{{$.NumberOfCommented}} participant</div>
```

Comment:
We should show `participants` instead of `participant` if `NumberOfCommented > 1`

Reply:
DONE